team.md

```diff
@@ -13,6 +13,10 @@ topLevel: true
 - Geoffrey Boushey (@boushey), Applications Programmer
 - Dave Leonard (@dartajax), User Support
 
+### Emeritus
```

Comment:
I think you'd want to say _Emeriti_ here as that's the plural. That will make it consistent with _Contributors_ above and _Sponsors_ below.